openedx/core/djangoapps/enrollments/api.py

```diff
@@ -494,6 +495,41 @@ def serialize_enrollments(enrollments):
     return _data_api().serialize_enrollments(enrollments)
 
 
+def is_enrollment_valid_for_proctoring(username, course_id):
```

Comment:
@Zacharis278 I moved the logic here so that it's more easily accessible - I haven't refactored the function you linked earlier, but it can probably pull this in as well. Let me know if this makes sense.

Reply:
Can we point that other function here? I'm worried it will be forgotten and trip someone up later on.

Reply:
Agreed, that function is refactored now + the tests updated to account for it